lib/order.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'twilio'
+require_relative 'menu'
+
+class Customer
+  attr_reader :order, :menu,
+
+  def initialize(menu=Menu.new)
+    @order = {}
+    @menu = menu
+  end
+
+  def add(food, quantity=1)
+    if @menu.foods.key?(food)
+      order.store(food, quantity)
+      "#{quantity}x #{food}(s) have been added to your order."
+    else
+      fail 'Sorry, that item isn\'t available on our menu.'
+    end
+  end
+
```

Comment:
could do with refactoring to separate responsibilities and eliminate code smells.